adoc/admin-disaster-recovery-restore-master-node.adoc

```diff
@@ -0,0 +1,108 @@
+= Recover Lost Master Node
+This chapter describes how to recover {productname} master nodes.
+
+== Replace Single Master Node
```

Comment:
Besides single master node case, we also needs to have unmatched etcd quorum case (whole cluster broken) like you have done in caasp-test-cases repo.